### PR TITLE
add stata pygments to stats cheatsheet

### PR DESCRIPTION
**Pull request by natashawatkins**
While we're at it, I realised stata has highlighting as well...

### CODE REVIEW

**Comment by mmcky** [comment]
@natashawatkins nice work. Thanks.

**Review by mmcky — APPROVED** [review]

**Comment by mmcky** [comment]
@natashawatkins 

When I compile this branch I get the following error message:

```
/home/mmcky/repos-collab/QuantEcon.cheatsheet/stats-cheatsheet.rst:19: WARNING: Pygments lexer name 'stata' is not known
```

Do you have to install the stata highlighter for pygments?

File: _themes/cheatsheet/support.py [hunk 1]
```diff
@@ -2,3 +2,4 @@
 from pygments.token import Keyword, Name, Comment, String, Error, \
      Number, Operator, Generic, Whitespace, Punctuation, Other, Literal
 from pygments.lexers.r import SLexer
+from pygment.lexers.stata import StataLexer
```
Comment (mmcky) [inline _themes/cheatsheet/support.py]: should this be ``pygments``. I can't find this lexer on my default anaconda installation

Comment (natashawatkins) [reply _themes/cheatsheet/support.py]: Yes it is! Oops, for some reason it still works on mine... weird.

**Comment by natashawatkins** [comment]
@mmcky this is fixed - want to try to compile again?

**Comment by mmcky** [comment]
still getting an import error. Can you try running ``make clean`` and then ``make html`` to see if your compilation is error free?

**Comment by natashawatkins** [comment]
Well the highlighting in the cheatsheet is still working fine

I get these but they also appear with the old master (before merging)

screen shot 2017-05-26 at 9 02 41 am

**Comment by mmcky** [comment]
Ok. So this requires ``pygments=2.2.0`` which is newer than base anaconda.
https://bitbucket.org/birkenfeld/pygments-main/src/tip/CHANGES?fileviewer=file-view-default
we will need to update the versions on the build server for this to compile properly.

**Comment by mmcky** [comment]
pygments on the build server is now ``2.2.0``.

**Pull request merged**